knowledge/responses/what_about_protein.md

```diff
@@ -1,17 +1,18 @@
 
-To start, there are several athletes who have sustained or increased their performances under vegan diets. Several athletic and strength world record holders completed their records as vegans, and more athletes, powerlifters, and bodybuilders are seeing continued improvements after switching to a vegan diet. This helps prove that **a pure vegan diet can allow enough protein to maintain an athletic lifestyle.** Although some people would point out that these athletes only get enough protein by supplementing it, it is important to note that supplements are nothing new for athletes. To the average person it can seem unnecessary to convert to a plant based diet just to resort to supplements, and it is true that this would be inconvenient, but in reality, vegan athletes supplement because they are athletes, not because they are vegan. All athletes supplement. For the average person **a proper vegan diet would not necessitate protein supplementation.**
+To start, there are several athletes who have sustained or increased their performances under vegan diets. Several athletic and strength world record holders completed their records as vegans, and more athletes, powerlifters, and bodybuilders are seeing continued improvements after switching to a vegan diet [1], proving that a vegan diet provides sufficient protein to maintain an athletic lifestyle. Although some may point out that these athletes only get enough protein through supplementaiton, it is important to note that supplements are nothing new for athletes. In reality, vegan athletes supplement because they are athletes, not because they are vegan. All athletes supplement [2]. For the average person, a proper vegan diet would not necessitate protein supplementation.
 
-A lot of us have been conditioned to think protein and animals are inseparable for so long that it is easy to miss out on the protein that's readily available in virtually all plant based sources. Although eating complimentary foods together can grant a complete protein, such as beans and rice, **several whole plant food sources have a complete amino acid profile all on their own.** Such foods include: quinoa, buckwheat, soy, various types of seeds, and more. Just looking at the amino acid profiles of many popular whole plant foods gives a clear view of readily available protein in vegan diets.
+A lot of us have been conditioned to think protein and animals are inseparable for so long that it is easy to miss out on the protein that's readily available in virtually all plant-based sources. Although eating complementary foods together can grant a complete protein, such as beans and rice, several whole plant food sources have a complete amino acid profile all on their own. Such foods include: quinoa, buckwheat, soy, various types of seeds, and more [3]. Looking at the amino acid profiles of many whole plant foods gives a clear view of readily available protein in vegan diets.
```

Comment:
I think it would be nice to touch on the fact that protein deficiency is an extremely difficult condition to "get" if you eat enough food. https://theconversation.com/the-protein-gap-nutritional-sciences-biggest-error-76202. Here's an article on it, put simply, humans have long been taught that they need more protein than they actually do. Excess protein is converted into fat and stored as fat in the body - there are benefits to consuming more protein than we need to - in fact there are bad consequences to it: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4045293/#B20.

Reply:
Also possibly adding in that while protein deficiencies are virtually non-existent, 90% of the US population fail to meet their fiber requirements: https://pubmed.ncbi.nlm.nih.gov/22649260/.

Reply:
Also a very prevalent myth is that plants do not have the full amino acid profile. I would replace "complete protein" with "full essential amino acid profile" for clarity. It is absolutely false but many people are taught this in school, so it is important to debunk. If we accept wikipedia as a source: https://en.wikipedia.org/wiki/Essential_amino_acids_in_plant_food, otherwise: https://www.ahajournals.org/doi/full/10.1161/01.CIR.0000018905.97677.1F.

Reply:
Addressed first comment. Did not add the fibre comment. Addressed full essential amino acid profile.